README.md

```diff
@@ -24,67 +24,31 @@ git clone git@github.com:PDFTron/webviewer-salesforce.git
 cd webviewer-salesforce
 ```
 
-3. Extract `WebViewer.zip`, `cd` to the directory the contents were extracted
```

Comment:
Can we keep steps 3,4,5 as optional steps in case they want to upgrade/downgrade to a different version manually?